src/work-allocation/containers/task-assignment-choose-role/task-assignment-choose-role.component.spec.ts

```diff
@@ -4,7 +4,7 @@ import { ActivatedRoute, convertToParamMap, Router } from '@angular/router';
 import { RouterTestingModule } from '@angular/router/testing';
 import { SessionStorageService } from '@hmcts/ccd-case-ui-toolkit/dist/shared/services';
 import { ExuiCommonLibModule } from '@hmcts/rpx-xui-common-lib';
-import { TASK_ROLES } from '../../../../api/workAllocation2/constants/task-roles.mock.data';
+import { TASK_ROLES } from '../../../../api/workAllocation/constants/task-roles.mock.data';
```

Comment:
Should not be using api folder- should replicate on src

Reply:
Scope of "Removing MVP" is not finetuning previously developed code , explained in detail on another comment.